src/statsmodel.jl

```diff
@@ -106,12 +106,50 @@ function formula end
 formula(m::TableStatisticalModel) = m.mf.f
 formula(m::TableRegressionModel) = m.mf.f
 
+"""
+    termnames(model::StatisticalModel)
+    termnames(term::AbstractTerm)
+
+Return the names associated with terms associated with a model.
+
+For models with only continuous predictors, this is the same as
```

Comment:
Only regression models have `responsename`:
```suggestion
For regression models with only continuous predictors, this is the same as
```